packages/wdio-devtools-service/src/index.ts

```diff
@@ -222,7 +222,8 @@ export default class DevToolsService implements Services.ServiceInstance {
 
         this._target = await this._puppeteer.waitForTarget(
             /* istanbul ignore next */
-            (t) => t.type() === 'page')
+            // @ts-ignore
+            (t) => t._targetInfo.browserContextId != null)
```

Comment:
Would this allow us to remove the `@ts-ignore`?
```suggestion
            (t) => t['_targetInfo'].browserContextId)
```

Reply:
This works just as well indeed.